src/main/java/de/unistuttgart/iste/sqa/pse/sheet11/presence/calculator/Calculator.java

```diff
@@ -28,16 +28,14 @@
 	 */
 	public int add(final int summand1, final int summand2);
```

Comment:
If we want a showcase, we can add the `pure` JML statement to both operations.
`public int /*@ pure @*/ add...`